### PR TITLE
Odh 7075 - new chapter for IDEs

### PR DESCRIPTION
**Pull request by MelissaFlinn**
New guide - Working in your IDE

### CODE REVIEW

File: modules/accessing-your-workbench-ide.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+To access a workbench IDE, use the link provided in the {productname-short} interface.
+
+.Prerequisite
+* You have created a data science project and a workbench.
+
+.Procedure
+
+. From the {productname-short} dashboard, click *Data Science Projects*. 
+. Click the name of the project that contains the workbench.
+. If the status of the workbench is *Running*, skip to Step 5.
```
Comment (aduquett) [inline modules/accessing-your-workbench-ide.adoc:15]: There is no step 5

File: modules/working-in-code-server.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::[]
+
+
+== Installing extensions with code-server
+
+With the code-server workbench image, you can customize your code-server environment by adding extensions that add new languages, themes, debuggers, and connect to additional services. You can also enhance the efficiency of your data science work with extensions for syntax highlighting, auto-indentation, and bracket matching.
```
Comment (aduquett) [inline modules/working-in-code-server.adoc:34]: A bit confusing. Maybe something like: 
_With the code-server workbench image, you can customize your code-server environment by using extensions to add new languages, themes, and debuggers, and to connect to additional services._

File: modules/working-in-code-server.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+The *Data Science Projects* page opens.
+. Click the name of the project containing the code-server workbench you want to start.
++
+A project details page opens.
+. Click the *Workbenches* tab.
+. Click the toggle in the *Status* column for the relevant workbench to start a workbench that is not running.
++
+The status of the workbench that you started changes from *Stopped* to *Running*.
+. After the workbench has started, click *Open* to open the workbench notebook.
+. In the Activity Bar, click the Extensions icon. (image:images/codeserver-extensions-icon.png[The Extensions icon])
```
Comment (aduquett) [inline modules/working-in-code-server.adoc:63]: Do we usually reference icons in this order?
In the Activity Bar, click the Extensions (image:images/codeserver-extensions-icon.png[The Extensions icon]) icon.

File: modules/working-in-rstudio-server.adoc
Comment (aduquett) [inline modules/working-in-rstudio-server.adoc:1]: Is this included upstream?

File: modules/working-in-rstudio-server.adoc [hunk 4]
```diff
@@ -1,88 +1,55 @@
...
 
-Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete.
-Red{nbsp}Hat does not recommend using them in production.
-These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
 
-For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
-====
+== Building the RStudio Server notebook images
 
 To use the *RStudio Server* and *CUDA - RStudio Server* notebook images, you must first build them by creating a secret and triggering the BuildConfig, and then enable them in the {productname-short} UI by editing the `rstudio-rhel9` and `cuda-rstudio-rhel9` image streams.
```
Comment (aduquett) [inline modules/working-in-rstudio-server.adoc:54]: Should this be BuildConfig ?

File: modules/working-in-code-server.adoc (hunk 3 above)
Comment (MelissaFlinn) [reply modules/working-in-code-server.adoc]: this is in the code-server UI. I made a minor fix to the text.

**Pull request merged**